pkg/meta/tkv.go

```diff
@@ -3323,6 +3323,16 @@ func (m *kvMeta) DumpMeta(w io.Writer, root Ino, keepSecret, fast, skipTrash boo
 		tree = m.snap[root]
 		if !skipTrash {
 			trash = m.snap[TrashInode]
+			if trash == nil {
```

Comment:
When this happen? It's OK to have no trash.

Reply:
Uniform behavior is for easier testing.